pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -106,11 +106,11 @@ func (ci *CloudInfo) getFlavor(flavorName string) (*flavors.Flavor, error) {
 
 func (ci *CloudInfo) getNetwork(networkName string) (*networks.Network, error) {
 	if networkName == "" {
-		return &networks.Network{}, nil
+		return nil, nil
 	}
 	networkID, err := networkutils.IDFromName(ci.clients.networkClient, networkName)
 	if err != nil {
-		if errors.Is(err, gophercloud.ErrResourceNotFound{}) {
+		if _, ok := err.(*gophercloud.ErrResourceNotFound); ok {
```

Comment:
this seems backwards... we should be moving towards errors.Is/As for error assertions from interface assertions.